src/account_manager.rs

```diff
@@ -751,17 +751,18 @@ impl AccountManager {
             HashMap::<String, u32>::with_capacity(local_device_ids.len());
 
         let signature_valid_on_each_signed_pre_key = true;
+        let default_device_id: u32 = (*DEFAULT_DEVICE_ID).into();
         for local_device_id in
-            std::iter::once(DEFAULT_DEVICE_ID).chain(local_device_ids)
+            std::iter::once(default_device_id).chain(local_device_ids)
```

Comment:
Should we instead make sure that `local_device_ids` is also the strong type, from the get_sub_device_sessions call? Or don't we control that trait?